core/embed/extmod/modtrezorcrypto/modtrezorcrypto-crc.h

```diff
@@ -26,9 +26,15 @@
 /// package: trezorcrypto.crc
 
 /// def crc32(data: bytes, crc: int = 0) -> int:
-///     """
-///     Computes a CRC32 checksum of `data`.
-///     """
+///    """
+///    Computes a CRC32 checksum of `data`.\n
```

Comment:
yeah you can't insert empty lines sorry

Reply:
@matejcik
Well, the VSCode understands the "\n" and displays the documentation hint correctly formatted. 

But if you do not want to have the "\n" in code, I'll remove them.

Reply:
nah it's good, i just thought that you need to push the newline to the generated files :woman_shrugging:

Reply:
@matejcik 
After discussion with @andrewkozlik, I have decided to not use the "\n" in the comments, but rather change the `make gen` script so that it allows the addition of empty lines. Once this pull request is merged, I will create a second PR for the changes in the gen script.

Reply:
it's a reasonable idea but last time i tried I got stuck on that quest for several hours with no success. so maybe not the best use of your time.

Reply:
Well, I have it ready :)